flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/HsFileDataManager.java

```diff
@@ -153,14 +153,18 @@ public synchronized void run() {
 
     /** This method only called by result partition to create subpartitionFileReader. */
     public HsDataView registerNewSubpartition(
-            int subpartitionId, HsSubpartitionViewInternalOperations operation) throws IOException {
+            int subpartitionId,
```

Comment:
We should change this method name to `registerNewConsumer`.

Actually, I think we should change it for all `HsResultPartition`, `HsMemoryDataManager`, `HsSubpartitionMemoryDataManager`, and even rename `HsSubpartitionView` and `HsSubpartitionViewInternalOperations`. We can leave the interface `ResultSubpartitionView` as is, which is shared by different shuffle implementations.

Reply:
Rename `HsSubpartitionView ` to `HsSubpartitionConsumer`, `HsSubpartitionViewInternalOperations ` to `HsSubpartitionConsumerInternalOperations `, `HsSubpartitionMemoryDataConsumer` to `HsSubpartitionConsumerMemoryDataManager`.